recipes-wpe/wpeframework/wpeframework-plugins_git.bb

```diff
@@ -50,6 +50,11 @@ WPE_COMPOSITOR_DEP ?= "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'weste
 WPE_COMPOSITOR_EXTRAFLAGS ?= ""
 WPE_COMPOSITOR_HARDWARE_READY ?= "0"
 
+WPE_POWER_AUTOSTART ?= "true"
+WPE_POWER_GPIOPIN ?= ""
+WPE_POWER_GPIOTYPE ?= ""
+WPE_POWER_DEP = "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/egl', 'broadcom-refsw', 'broadcom-refsw', '', d)}"
```

Comment:
this is not correct place to do SOC specific changes. They should be done in broadcom layer via a bbappend where it would be simple assignment instead of complex conditional. here just set it to ```?= ""```

Reply:
I agree on master we could that, unfortunately on the morty/krogoth branches Broadcom does not apply that in their 17.x / 18.x releases and we'll need that logic there. Without that it would get very confusing for new comers to RDKV to create a working config for their setup.